channels/applicationinsights-channel-js/src/Interfaces.ts

```diff
@@ -115,6 +115,12 @@ export interface ISenderConfig {
      * @since 3.0.4
      */
     alwaysUseXhrOverride?: boolean;
+
+      /**
+     * [Optional] Either an array or single value identifying the requested TransportType type that should be used.
+     * This is used during initialization to identify the requested send transport, it will be ignored if a httpXHROverride is provided.
+     */
+      transports?: number | number[];
```

Comment:
Actually, looking at the PostChannel we also have an `unloadTransports` that we should add here as well.